website/views/pages/device-management.ejs

```diff
@@ -833,41 +823,41 @@
           </div>
         </div>
 
-        <div purpose="button-row" style="margin-top: 32px;" class="d-flex justify-content-center align-items-center mx-auto">
-          <a purpose="cta-button" href="/pricing">Compare all features</a>
-          <animated-arrow-button  href="/testimonials">What are people saying?</animated-arrow-button>
+        <div purpose="button-row" style="margin-top: 40px;" class="d-flex flex-sm-row flex-column justify-content-center align-items-center mx-auto">
+          <a purpose="cta-button" href="/register">Try it yourself</a>
+          <animated-arrow-button href="/contact">Talk to an engineer</animated-arrow-button>
         </div>
       </div>
       <%/* Shorten the feedback loop section */%>
       <div purpose="page-section">
         <div purpose="feature" class="d-flex flex-md-row flex-column-reverse justify-content-between mx-auto align-items-center">
           <div purpose="feature-text" class="d-flex flex-column">
             <h2>Shorten the feedback loop</h2>
-            <p>Spend less time debugging whether changes actually happened.</p>
+            <p>Built for speed and transparency. No more waiting for closed systems to catch up.</p>
             <div purpose="checklist" class="flex-column d-flex">
               <p><a href="/orchestration">Get near-instant visibility</a> into your Apple, Linux, and Windows devices from a single, open platform.</p>
-              <p>Verify security changes in seconds — every action is tracked in Fleet’s UI or your logs.</p>
-              <p>Avoid tickets with real-time confirmation of patch and config changes.</p>
+              <p>Verify changes in real time, with full audit logs.</p>
+              <p>Confidently resolve issues before users notice.</p>
             </div>
           </div>
           <div purpose="feature-image">
             <img alt="A laptop using Fleet to accurately display the status of MDM configurations" src="/images/mdm-shorten-feedback-loop-528x377@2x.png">
           </div>
         </div>
       </div>
-      <%/* Mordern change management section */%>
```

Comment:
Allen:

Let's move this lower down. It's more of a benefit than a reason for moving to Fleet, IMO.